client/src/views/NewProject.vue

```diff
@@ -74,17 +77,15 @@ export default {
     },
     async submitProject() {
       if (this.isNameTaken(this.form.projectName) && this.form.name) {
-        alert(`The name: ${this.form.projectName} is taken!`);
+        alert(`The name: ${this.form.projectName} is taken!`); // eslint-disable-line
       } else {
         const accessToken = await this.$auth.getTokenSilently();
-        console.log(accessToken);
-
         const project = {
           members: JSON.stringify([this.$store.state.currentUser.id]),
           name: this.form.projectName,
           links: this.form.links,
```

Comment:
Make sure you're using `JSON.stringify` on any array of string you're shipping off to the db. The arr should look like: `'["item1", "item2", "item3"]'` when in the database.